ur_robot_driver/include/ur_robot_driver/hardware_interface.hpp

```diff
@@ -45,10 +44,10 @@
 
 // ROS
 #include "rclcpp/macros.hpp"
+#include "rclcpp_lifecycle/state.hpp"
 
 using hardware_interface::HardwareInfo;
 using hardware_interface::return_type;
```

Comment:
I propose to remove this using. There are only a few places where they are useful, and it is clearer when full type reference is written-out.

```suggestion
```

Reply:
d4a1834